app/v1/controllers/requestParsers/ListLossClaimsParser.scala

```diff
@@ -19,7 +19,7 @@ package v1.controllers.requestParsers
 import javax.inject.Inject
 import uk.gov.hmrc.domain.Nino
 import v1.controllers.requestParsers.validators.ListLossClaimsValidator
-import v1.models.domain.{ClaimType, TypeOfLoss}
```

Comment:
How come the name "ClaimType" was changed?

Reply:
There was already a data structure for that information, I just swapped to using it.